src/chrome/content/rules/perspectives-project.org.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="perspectives-project.org (mismatched)" default_off="mismatched"> 
```

Comment:
perspectives-project.org is in [WordPress-blogs.xml](https://github.com/EFForg/https-everywhere/blob/master/src/chrome/content/rules/WordPress-blogs.xml).